src/controllers/dynakube/version/updater.go

```diff
@@ -139,6 +140,39 @@ func updateVersionStatus(
 	return nil
 }
 
+func updateVersionStatusForTenantRegistry(
+	ctx context.Context,
+	target *dynatracev1beta1.VersionStatus,
+	imageUri string,
+	imageVersionFunc ImageVersionFunc,
+	dockerCfg *dockerconfig.DockerConfig,
+) error {
+	ref, err := reference.Parse(imageUri)
+	if err != nil {
+		return errors.WithMessage(err, "failed to parse image uri")
+	}
+
+	log.Info("updating image version info for tenant registry image",
+		"image", imageUri,
+		"oldImageID", target.ImageID,
+		"oldVersion", target.Version)
+
+	if taggedRef, ok := ref.(reference.NamedTagged); ok {
+		imageVersion, err := imageVersionFunc(ctx, imageUri, dockerCfg)
+		if err != nil {
+			log.Error(err, "failed to get image digest, falling back to tag")
```

Comment:
This log is wrong as you actually fail to get the image version and not the image digest